dev/com.ibm.websphere.appserver.features/visibility/public/concurrent-2.0/com.ibm.websphere.appserver.concurrent-2.0.feature

```diff
@@ -15,11 +15,10 @@ Subsystem-Name: Jakarta EE Concurrency 2.0
   com.ibm.websphere.appserver.containerServices-1.0, \
   com.ibm.websphere.appserver.contextService-1.0, \
   com.ibm.websphere.appserver.eeCompatible-9.0, \
-  io.openliberty.jakarta.concurrency-2.0, \
   com.ibm.websphere.appserver.org.eclipse.microprofile.contextpropagation-1.0; ibm.tolerates:="1.1"
 -bundles=\
-  com.ibm.websphere.javaee.concurrent.1.0; location:="dev/api/spec/,lib/"; apiJar=false, \
-  com.ibm.ws.concurrent, \
+  io.openliberty.jakarta.concurrency.2.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:2.0.0-RC2",\
+  com.ibm.ws.concurrent.jakarta, \
   com.ibm.ws.javaee.platform.defaultresource, \
   com.ibm.ws.resource
 kind=beta
```

Comment:
FYI, I asked @aguibert to rename this feature to `io.openliberty.concurrent-2.0`

Reply:
I'll wait for this PR to go in before I rename